Czaple/Attractions/Models/BoardAlert.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BoardAlert.swift
+//  Czaple
+//
+//  Created by Filip Trzaska on 05/12/2019.
+//  Copyright © 2019 Filip Trzaska. All rights reserved.
+//
+
+import UIKit
+
+protocol BoardAlerting {
+    func boardAlertAction(controller: UIViewController)
+}
+
+final class BoardAlert: BoardAlerting {
+    let boardProvider: BoardProviding
+    let boardAudioPlayer: BoardAudioPlayerProtocol
+    
+    init(boardProvider: BoardProviding, boardAudioPlayer: BoardAudioPlayerProtocol) {
+        self.boardProvider = boardProvider
+        self.boardAudioPlayer = boardAudioPlayer
+    }
+    
+    func boardAlertAction(controller: UIViewController) {
```

Comment:
You should not pass viewController here, rather return this alert controller in method and then present it in place when you call that method :)

Reply:
It's because this method should only focus on creating this alert, it will be more flexible if you let present it by someone else :)